content/pt-br/shift-left.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Shift Left
+status: Completed
+category: conceito
+tags: ["metodologia", "", ""]
+---
+
+## O que é
+
+Esquerda em Shift Left se refere aos estágios iniciais no ciclo de desenvolvimento de software,
+pensando no ciclo como uma linha onde os estágios são executados da esquerda para a direita.
+Shift left é a pratica de implementar testes, segurança e outras praticas de desenvolvimento
+no início do ciclo de desenvolvimento de software e não no final.
+
+Embora originalmente usado para se referir ao processo de teste de forma antecipada,
+Shift Left agora também pode ser aplicado para outros aspectos do desenvolvimento de software e [DevOps](/devops/), como segurança e implementação (deployment).
+
+## Problema relacionado
+
+Problemas de segurança, erros e defeitos de software podem ser mais idificeis e caros para serem corrigidos
+se eles forem descobertos de forma tardia durante o ciclo de desenvolvimento ou após a implementação,
+principalmente se o software já estiver implementado em produção.
+
+## Como isso ajuda
+
+Adotar a mentalidade de Shift Left para o desenvolvimento de software,
+as equipes podem implementar testes e segurança durante o ciclo de desenvolvimento.
+Pela responsabilidade por testes e segurança ser compartilhada pela equipe de desenvolvimento
+— dos engenheiros de software, qualidade até operações —
+todos tem seu papel em garantir a estabilidade e segurança de uma aplicação.
+
+Além disso, atuar de forma Shift Left habilita a melhoria contínua e
+está em linha com uma abordagem mais [agile](/agile-software-development/) do que cascata ao desenvolvimento.
+Times podem fazer pequenas interações e identificar processos de forma antecipada.
+Essa abordagem permite que egenheiros adotem práticas de segurança e desenvolvimento seguro
+nos estágios inicialis, assim como as fases de design e arquitetura.
+Testando durante o ciclo de desenvolvimento, diminui o tempo necessário de testes antes de uma release de software.
+
+Muitas ferramentas de software e soluções SaaS ajudam a mudar (shift) essas práticas para a esquerda (left).
+
+Entreteanto, shift left também pode ser implementado através da melhoria de processos e mudanças culturais junto com o time.
```

Comment:
"Entretanto" instead of "Entreteanto".

"dentro de uma equipe." instead of "junto com o time" sounds better to me.